src/xml/xmlNode.ts

```
@@ -436,19 +447,31 @@ export const XmlNode = {
+        if (!node) return null;
+        return (node.childNodes || []).filter(child => child.nodeName === childName);
+    },
+
+    findChildrenByNameDeep(node: XmlNode, childName: string): XmlNode[] {
+        if (!node) return null;
+
+        const found = this.findChildrenByName(node, childName);
+
+        const deep = (node.childNodes || []).flatMap(child => this.findChildrenByNameDeep(child, childName));
+
+        return [...found, ...deep].flatMap(n => n).filter(n => n);
```

Biome (1.9.4)

[error] Avoid unnecessary callback in flatMap call.

You can just use flat to flatten the array.
Safe fix: Replace unnecessary flatMap call to flat instead.


(lint/correctness/noFlatMapIdentity)